spark/src/main/java/org/apache/iceberg/actions/BaseSparkAction.java

```diff
@@ -47,9 +87,9 @@
   protected Dataset<Row> buildValidDataFileDF(SparkSession spark, String tableName) {
     JavaSparkContext context = new JavaSparkContext(spark.sparkContext());
     Broadcast<FileIO> ioBroadcast = context.broadcast(SparkUtil.serializableFileIO(table()));
-    String allManifestsMetadataTable = metadataTableName(tableName, MetadataTableType.ALL_MANIFESTS);
 
-    Dataset<ManifestFileBean> allManifests = spark.read().format("iceberg").load(allManifestsMetadataTable)
+    Dataset<ManifestFileBean> allManifests = loadMetadataTable(spark, tableName, table().location(),
+        MetadataTableType.ALL_MANIFESTS)
```

Comment:
If we do a static import for `ALL_MANIFESTS`, will it fit on one line?

Reply:
If not, we can make a var `String tableLocation = table().location()` if we need to reduce the length further.